src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    public String getColor() {
+        return color;
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    public int getNumber() {
+        return number;
+    }
+
+    public void setNumber(int number) {
+        this.number = number;
+    }
+
+    public int getRandomNumber() {
+        int num = new Random().nextInt(100);

```

Comment:
You must not use the Random in Models, 
You should use it in Services like (ColorSupplier, Lottery)
```suggestion
```

All magic numbers, strings, symbols in your code should be constants.
[this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)

Reply:
Sorry, but I didn't understand what you meant when you wrote "All magic numbers, strings, symbols in your code should be constants."